### PR TITLE
Support templates in partitioned topics names

### PR DESCRIPTION
**Pull request by Pchelolo**
Bug https://phabricator.wikimedia.org/T219159

### CODE REVIEW

File: sys/partitioner.js [hunk 1]
```diff
@@ -21,6 +22,8 @@ class Partitioner {
         if (!options.partition_default) {
             throw new Error('No partition_default was provided to the partitioner');
         }
+
+        this._partitionedTopicTemplate = new Template(options.templates.partition_topic);
```
Comment (d00rman) [inline sys/partitioner.js]: There is no alternative provided here if `partition_topic_name` is provided directly (without a template), which means `options.templates` will be `undefined`, resulting in a run-time error.

Comment (Pchelolo) [reply sys/partitioner.js]: We drop support for partition_topic_name and it's checked in the beginning of the constructor.

Comment (d00rman) [reply sys/partitioner.js]: Ah ok, indeed I misread the `if`-block. In this case, let's please bump the minor version of CP.

Comment (Pchelolo) [reply sys/partitioner.js]: Done

Comment (d00rman) [reply sys/partitioner.js]: Heh, I was thinking of bumping the minor version, not the patch one, but ok. We shall do that too, one day, one day :)

**Comment by d00rman** [comment]
Published as [v0.9.2](https://github.com/wikimedia/change-propagation/tree/v0.9.2)

**Pull request merged**